src/state/bridge/hooks.ts

```diff
@@ -148,28 +148,36 @@ export function useBridgeActionHandlers(): {
 
 export function useBridgeFee(tokenAddress: string | undefined) {
   const { chainId, account, library } = useActiveWeb3React()
+  const { isHome } = useChain()
 
   return useAsyncMemo(async () => {
-    if (!chainId || !account || !library || !tokenAddress || !isMultiErc20ToErc677BridgeToken(tokenAddress)) return
+    if (!isHome || !account || !library || !tokenAddress || !isMultiErc20ToErc677BridgeToken(tokenAddress)) return
 
     const address = getHomeMultiErc20ToErc677BridgeAddress()
     const contract = getHomeMultiAMBErc20ToErc677Contract(address, library, account)
     const fee = await contract.getFee(HOME_TO_FOREIGN_FEE_TYPE_HASH, tokenAddress)
     return formatEther(fee)
-  }, [account, chainId, library, tokenAddress])
+  }, [isHome, account, chainId, library, tokenAddress])
```

Comment:
I think `chainId ` is not used here. Can  you go over the linting issues? Fix or put a ignore when needed

Reply:
I have removed chainId from the functions